bulk-delete.php

```diff
@@ -441,6 +448,28 @@ public function add_delete_posts_settings_panel() {
 		wp_enqueue_script( 'postbox' );
 	}
 
+	/**
+	 * Ajax call back function for getting taxonomies to load select2 options.
+	 *
+	 * @since 5.7.0
+	 */
+	public function load_taxonomy_term(){
+		$return = array();
+
+		$terms = get_terms( array(
+			'taxonomy' => $_GET['term'],
```

Comment:
Change the post variable to `taxonomy`, since it is actually a taxonomy and not term.